composer.json

```diff
@@ -100,7 +100,8 @@
         "phpstan/phpstan": "^1.4.7",
         "phpunit/phpunit": "^9.5.8",
         "predis/predis": "^1.1.9|^2.0.2",
-        "symfony/cache": "^6.0"
+        "symfony/cache": "^6.0",
+        "symfony/http-client": "^6.0"
```

Comment:
This should have also been added to the mailer composer.json, at least as a suggest.

Reply:
@GrahamCampbell that's already the case: https://github.com/laravel/framework/blob/9.x/src/Illuminate/Mail/composer.json#L41